rust/src/builder.rs

```diff
@@ -189,9 +189,6 @@ mod tests {
         }
         let a = b.finish();
         assert_eq!(5, a.len());
-        for i in 0..5 {
-            assert_eq!(&i, a.get(i as usize));
```

Comment:
Why remove those checks?

Reply:
`Buffer` now doesn't have a `get` method. In fact, `Builder` is now unused. I intend to remove it later after this PR and replace with an `ArrayBuilder`.